atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -97,21 +102,40 @@ def generate_auto_labels(self, auto_labels):
 
         for lbl in auto_labels:
             if lbl in self.labels:
-                self.log.info("label %s is set explicitly, not using generated value", lbl)
+                self.log.info("label %r is set explicitly, not using generated value", lbl)
                 continue
 
             if lbl in generated:
                 self.labels[lbl] = generated[lbl]
             else:
-                self.log.warning("requested automatic label %s is not available", lbl)
+                self.log.warning("requested automatic label %r is not available", lbl)
+
+    def add_renamed_labels(self, base_labels, df_labels, new_labels):
```

Comment:
I totally agree that "rename" is somewhat misleading and would rather go with alias.

I'd probably also advise emitting debug message if there is alias for a label that doesn't exist. From the docstring in `__init__` method I guess this situation is expected, but it still may be worth logging when it occurs, for easier debugging.

Otherwise this PR looks good.